io.catenax.shared.business_partner_number/2.0.0/BusinessPartnerNumber.ttl

```diff
@@ -0,0 +1,86 @@
+#######################################################################
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This work is made available under the terms of the
+# Creative Commons Attribution 4.0 International (CC-BY-4.0) license,
+# which is available at
+# https://creativecommons.org/licenses/by/4.0/legalcode.
+#
+# SPDX-License-Identifier: CC-BY-4.0
+#######################################################################
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#>.
```

Comment:
samm:meta-model 2.1.0 required in all new models.

Reply:
updated all resepective references stating to namepsace org.eclipse.esmf.samm